src/custom_connect.ts

```diff
@@ -0,0 +1,24 @@
+import net from 'net';
+import type http from 'http';
+import { promisify } from 'util';
+// import { countTargetBytes } from './utils/count_target_bytes';
+
+const asyncWrite = promisify(net.Socket.prototype.write);
+
+export const customConnect = async (socket: net.Socket, server: http.Server): Promise<void> => {
+    // countTargetBytes(socket, socket);
```

Comment:
Do we want to count bytes here? Also both `proxy-chain` v1 and v2 we don't count bytes when using `customResponseHandler`. How are users accounted for data transferred when using SERP?

Reply:
SERPs are billed per request, not per GBs

Reply:
Ah ok. Do we want to track bytes transferred anyway?

Reply:
If bytes are simple to count, I'd add it. If it's complicated, fuk it.